src/sidebar/store/modules/test/filters-test.js

```diff
@@ -113,6 +157,100 @@ describe('sidebar/store/modules/filters', () => {
       });
     });
 
+    describe('getFilter', () => {
+      it('returns the filter you ask for', () => {
```

Comment:
```suggestion
      it('returns the specified filter', () => {
```

Just a small stylistic suggestion.